spec/remote_spec.cr

```diff
@@ -20,7 +20,7 @@ describe Git::Remote do
   end
 
   it "shoud return push_url" do
-    repo.remotes["test_with_pushurl"].push_url.should eq("git://github.com/libgit2/pushlibgit2")
+    repo.remotes["test_with_pushurl"].push_url.should eq("git@github.com:libgit2/pushlibgit2")
```

Comment:
I'm not entirely sure why this changed, but it made the spec suite fail.

Reply:
Seems to have been only a local issue. Maybe libgit2 0.26 behaves different than 0.27